app/src/manager/admin.py

```diff
@@ -0,0 +1,71 @@
+from django.contrib import admin
+
+from .models import Block, Hotkey, Operator, Server, Subnet, SubnetSlot, ValidatorInstance
+
+
+# Admin Classes
+@admin.register(Subnet)
+class SubnetAdmin(admin.ModelAdmin):
+    list_display = ("name", "description", "get_operator_count")
+    search_fields = ("name",)
+    list_filter = ("operators",)
+
+    def get_operator_count(self, obj):
+        return obj.operators.count()
```

Comment:
this runs a query for every subnet. Do we really need it? I think what we need is columns which indicate:
- whether we are running a validator on mainnet
- whether we are running a validator on testnet
- short registration indicator: if a subnet is registered on mainnet and testnet as 12 and 147, this'll be sn12t147, if only mainnet, sn12, if only testnet, sn147, otherwise "-"
-